packages/node/src/indexer/unfinalizedBlocks.service.ts

```diff
@@ -88,7 +88,7 @@ export class UnfinalizedBlocksService {
     if (block) {
       await this.registerUnfinalizedBlock(
         block.block.header.number.toNumber(),
-        block.hash.toHex(),
```

Comment:
` block.hash` should be equal to `block.block.header.hash`, in packages/types/src/generic/Block.ts
![Uploading image.png…]()

Reply:
From my testing they are not the same. This was the reason that forks would happen when there wasn't actually a fork